services/distribution/src/test/resources/application.yaml

```diff
@@ -56,7 +56,7 @@ services:
     app-features:
       - label: isPlausibleDeniabilityActive
         value: ${PLAUSIBLE_DENIABILITY_ACTIVE:1}
-    supported-countries: ${SUPPORTED_COUNTRIES:DE}
+    supported-countries: ${SUPPORTED_COUNTRIES:DE,FR}
```

Comment:
+ Ireland, Netherlands, ... testing data is Ok to have larger set of default values